signer/tests/utxo_construction.rs

```diff
@@ -0,0 +1,255 @@
+use bitcoin::absolute::LockTime;
+use bitcoin::blockdata::opcodes;
+use bitcoin::hashes::Hash;
+use bitcoin::taproot::LeafVersion;
+use bitcoin::taproot::NodeInfo;
+use bitcoin::taproot::TaprootSpendInfo;
+use bitcoin::transaction::Version;
+use bitcoin::AddressType;
+use bitcoin::Amount;
+use bitcoin::OutPoint;
+use bitcoin::PubkeyHash;
+use bitcoin::ScriptBuf;
+use bitcoin::Sequence;
+use bitcoin::TapLeafHash;
+use bitcoin::Transaction;
+use bitcoin::TxIn;
+use bitcoin::TxOut;
+use bitcoin::Witness;
+use bitcoin::XOnlyPublicKey;
+use bitcoincore_rpc::json::ListUnspentResultEntry;
+use bitcoincore_rpc::RpcApi;
+use secp256k1::SECP256K1;
+use signer::utxo::DepositRequest;
+use signer::utxo::SbtcRequests;
+use signer::utxo::SignerBtcState;
+use signer::utxo::SignerUtxo;
+
+mod regtest;
+use regtest::Either;
+use regtest::Recipient;
+
+const SIGNER_ADDRESS_LABEL: Option<&str> = Some("signers-label");
+const DEPOSITS_LABEL: Option<&str> = Some("deposits");
+
+fn make_deposit(
```

Comment:
Nit: Just to clarify that this creates a deposit request.

```suggestion
fn make_deposit_request(
```